datasetinsights/stats/statistics.py

```diff
@@ -2,9 +2,12 @@
 from pathlib import Path
 
 import datasetinsights.constants as const
-from datasetinsights.data.simulation import MetricDefinitions, Metrics
-from datasetinsights.data.simulation.download import Downloader
-from datasetinsights.data.simulation.tables import SCHEMA_VERSION
+from datasetinsights.datasets.simulation import (
```

Comment:
datasetinsights.datasets.unity_perception?

Reply:
Will go in next PR.